code/game/objects/items/stacks/medical.dm

```diff
@@ -23,6 +23,9 @@
 		var/mob/living/carbon/human/H = M
 		var/obj/item/organ/external/affecting = H.get_organ(user.targeted_organ)
 
+		if(!affecting)
+			to_chat(user, SPAN_WARNING("What [user.targeted_organ]?"))
```

Comment:
`return` missed

Reply:
Fixed.

Reply:
<code>var/mob/living/carbon/human/victim = owner_ ? owner_ : owner // Used to keep and put into other's removed procs.</code>